### DISCUSSION TITLE
I can't get connection to work

### DISCUSSION THREAD

**Discussion by boba1234567890** [rustdesk/rustdesk#10259] [Q&A]
hi, I've spent many days and many long chats with GPT trying to get rust desk free to work 

I've hit the stage where I can get Hbbr and Hbbs to work and my phone (on mobile to simulate being offsite) but when i try to connect from my phone to my computer with a server (running hbbr and hbbs) on logs
 
"[2024-12-11 12:25:26.136781 +11:00] INFO [src\relay_server.rs:452] New relay request 7cfc734b-ee51-4435-ab78-bae71919292c from [::ffff:192.168.32.1]:51061"

 is printed but I can never get a successful connection 

Any help is appreciated
feel free to ask if you need more info 😄

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/763, go to [discord](https://discord.gg/nDceKgxnkV) please, search there first, some users might have asked similar questions. Usually nobody will help you with build or self-hosting here.
